js/jsonld.js

```diff
@@ -788,7 +788,7 @@ jsonld.normalize = function(input, options, callback) {
   }
 
   if('inputFormat' in options) {
-    if(options.inputFormat !== 'application/nquads') {
+    if((options.inputFormat !== 'application/nquads') || (options.inputFormat !== 'application/n-quads'))  {
```

Comment:
Shouldn't this be an `&&`? Otherwise it would seem that this compound conditional would always be true.

That being said, it would probably be cleaner to make a shallow copy of the options and if the `inputFormat` is `application/nquads` it should just be changed early to the correct `application/n-quads`. Then we only need to check for that in the code later.